tests/unit/amo/components/TestAddonReview.js

```diff
@@ -445,17 +445,17 @@ describe(__filename, () => {
       });
     });
 
-    describe('setDenormalizedReview', () => {
-      it('dispatches a setReview action', () => {
+    describe('setInternalReview', () => {
+      it('dispatches a setInternalReview action', () => {
         const review = {
           ...defaultReview,
           body: 'some body',
         };
-        actions.setDenormalizedReview(review);
+        actions.setInternalReview(review);
 
         sinon.assert.called(dispatch);
         const action = dispatch.firstCall.args[0];
-        expect(action.type).toEqual(SET_REVIEW);
+        expect(action.type).toEqual(SET_INTERNAL_REVIEW);
```

Comment:
You could clean this up while you're in here but no big deal.

```js
sinon.assert.calledWith(dispatch, setInternalReview(review));
```

Reply:
ah yes, makes sense